apps/webcert/src/components/Inputs/TextInput.tsx

```diff
@@ -13,13 +14,29 @@ interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
 }
 
 const TextInput = React.forwardRef<HTMLInputElement, Props>(
-  ({ label, id: controlledId, limit, className, hasValidationError, css, autoComplete, tooltip, showAsterix, required, ...props }, ref) => {
+  (
+    {
+      label,
+      italicLabel,
+      id: controlledId,
+      limit,
+      className,
+      hasValidationError,
+      css,
+      autoComplete,
+      tooltip,
+      showAsterix,
+      required,
+      ...props
+    },
+    ref
+  ) => {
     const uncontrolledId = useId()
     const id = controlledId ?? uncontrolledId
 
     return (
       <div>
-        {label && <FieldLabel id={id} label={label} tooltip={tooltip} required={showAsterix && required} />}
+        {label && <FieldLabel id={id} label={label} tooltip={tooltip} required={showAsterix && required} italic={italicLabel} />}
```

Comment:
On the sketches it looks like italic should be used when fields are disabled, in that case we could use the existing `disabled` prop. please check this with UX.